konan/konan.properties

```diff
@@ -652,7 +652,7 @@ linkerDynamicFlags.mingw_x64 = -shared
 linkerKonanFlags.mingw_x64 =-static-libgcc -static-libstdc++ \
   -Xclang -flto-visibility-public-std -Wl,--dynamicbase \
   -Wl,-Bstatic,--whole-archive -lwinpthread -Wl,--no-whole-archive,-Bdynamic \
-  -Wl,--defsym,__cxa_demangle=Konan_cxa_demangle
+  -Wl,--defsym,__cxa_demangle=Konan_cxa_demangle -lbcrypt
```

Comment:
I suggest to avoid adding this dependency unconditionally. You can add a separate property (e.g. `mimallocLinkerDependencies.mingw_x64`) and add it in `Linker.kt` if we're compiling with mimalloc enabled.